ios/NotifeeNotificationService/NotificationService.m

```diff
@@ -0,0 +1,47 @@
+//
+//  NotificationService.m
+//  NotifeeNotificationService
+//
+//  Created by Tyler Hammer on 1/11/24.
+//
+
+#import "NotificationService.h"
+#import "NotifeeExtensionHelper.h"
+#import "SendbirdNotificationHelper.h"
+
+@interface NotificationService ()
+
+@property (nonatomic, strong) void (^contentHandler)(UNNotificationContent *contentToDeliver);
+@property (nonatomic, strong) UNMutableNotificationContent *bestAttemptContent;
+
+@end
+
+@implementation NotificationService
+
+- (void)didReceiveNotificationRequest:(UNNotificationRequest *)request withContentHandler:(void (^)(UNNotificationContent * _Nonnull))contentHandler {
+    self.contentHandler = contentHandler;
+    self.bestAttemptContent = [request.content mutableCopy];
+  
+  NSLog(@"%@", request);
+  NSLog(@"Logging Notification");
+  
+  [SendbirdNotificationHelper setAppGroup:@"group.sample.chat.react-native"];
```

Comment:
You can add App Groups in `Signing & Capabilities > TARGETS > notifications > +Capability > App Groups`

![image](https://github.com/sendbird/sbm-react-native-sample/assets/26326015/15fa4d2e-2077-4843-af7d-44b712e3b2c2)